algorithms/C/maths/decimal_to_binary.c

```diff
@@ -0,0 +1,30 @@
+/* This program is used to convert decimal numbers to their binary equivalent
+   Time Complexity : O(logn)
+   Auxiliary Space: O(1)
+*/
+
+#include <stdio.h>
+void decimal_to_binary(int number)
```

Comment:
Suggestion: it would be much better if the function just returned the converted number ( maybe in form of a string )  and the caller take responsibility of using that value according to its needs.